Detectors/ITSMFT/MFT/workflow/src/TrackerSpec.cxx

```diff
@@ -263,24 +263,34 @@ void TrackerDPL::run(ProcessingContext& pc)
     }
 
   } else {
+    LOG(info) << "Field is off! ";
     std::vector<std::vector<o2::mft::ROframe<TrackLTFL>>> roFrameVec(mNThreads); // One vector of ROFrames per thread
+    LOG(info) << "Reserving ROFs ";
```

Comment:
could you change these logs to debug level?

Reply:
done, thanks.